### PR TITLE
Implement Google Analytics (Fixes dockerfile/ghost#14)

### PR DESCRIPTION
**Pull request by zedtux**
This pull request add the Google Analytics.

A user can then pass the `GA_ID` within the `docker run` which will then insert it on start.

### CODE REVIEW

File: start.bash [hunk 1]
```diff
@@ -35,6 +35,9 @@ if [[ -d "$OVERRIDE/$THEMES" ]]; then
   done
 fi
 
+# Insert the Google Analytics ID
+sed -i 's/UA-XXXXXXX-XX/'$GA_ID'/' /ghost/content/themes/casper/default.hbs
```
Comment (Bilge) [inline start.bash:39]: This should just be `"s/UA-XXXXXXX-XX/$GA_ID/"` or even `s/UA-XXXXXXX-XX/"GA_ID"/`.

**Comment by Bilge** [comment]
I don't see the point of this PR. If you want to customize your theme that's what the override directory is for.

**Comment by zedtux** [comment]
Up to you... Non tech people would like it but if you don't ... then they will not have it. Fine.

**Comment by Bilge** [comment]
You can always maintain a fork that implements this if you think it provides value.

**Pull request closed without merging**